packages/OverflowMenu/src/OverflowMenu.js

```diff
@@ -71,11 +74,12 @@ function OverflowMenu(props) {
 
   const handleOpenMenu = () => {
     setIsOpen(true);
-    // https://github.com/acl-services/paprika/issues/316
-    // Todo Should focus the first item via an onAfterOpen event callback in popover
-    setTimeout(() => {
-      focusAndSetIndex(0);
-    }, 250);
+
+    if (document.querySelector("html").getAttribute("data-whatinput") === "keyboard") {
+      setTimeout(() => {
```

Comment:
it might safet if you do `window.requestAnimationFrame(() => {
  // or does this get triggers when the overflow menu is still visible?
})`

Reply:
@nahumzs are you suggesting that I change it to this?

```
  const handleOpenMenu = () => {
    setIsOpen(true);

    if (document.querySelector("[data-whatinput]").dataset.whatinput === "keyboard") {
      setTimeout(() => {
        window.requestAnimationFrame(() => {
          focusAndSetIndex(0);
        });
      }, 250);
    }
  };
```

Reply:
ohh you don't need the settimeout if you are using the requestAnimationFrame, but I'm suspecting that the real problem is that we don't have a callback for the popover which has animation to be called once the animation finished :/.

In that case requestAnimationFrame will be not sufficient.

Reply:
Yeah replacing `setTimeout` with `requestAnimationFrame` does not work.